packages/medusa/src/api/routes/store/variants/get-variant.ts

```diff
@@ -113,8 +112,19 @@ export default async (req, res) => {
   res.json({ variant })
 }
 
-export class StoreGetVariantsVariantParams extends PriceSelectionParams {
+export class StoreGetVariantsVariantParams
+  extends PriceSelectionParams
```

Comment:
todo: I would say that `PriceSelectionParams` should extends `FindParams` wdy?

Reply:
It could technically expand the `FindParams` but I think we're then bundling things together for convenience rather than what makes sense semantically. Since `FindParams` is not semantically linked to `PriceSelectionParams` it wouldn't make much sense to enforce the parameters if you extend `PriceSelectionParams` since we're then forcing unrelated fields into the object.

I see it from an "all endpoints needs these params" perspective, but maybe there are better ways of expressing this with types, perhaps using mixins?

Reply:
The `FindParams` limit and offset are optional, I don't think it is a problem in that case that they are bundled together. If it was some non optional deps then I would have been aggreed with you. wdyt?